docs/docs-components/Header.js

```diff
@@ -46,10 +52,10 @@ function SettingsDropdown({
     return setTextDirection(textDirection === 'rtl' ? 'ltr' : 'rtl');
   };
 
-  const onChangeSandpackVisibility = () => {
-    trackButtonClick('Toggle Sandpack visibility', sandpack);
+  const onChangeDevExampleMode = () => {
+    trackButtonClick('Toggle Sandpack visibility', devExampleMode);
     closeDropdown();
-    return setSandpack(sandpack === 'enabled' ? 'disabled' : 'enabled');
+    return setDevExampleMode(devExampleMode === 'sandpack' ? 'classic' : 'sandpack');
```

Comment:
Rather than always showing the dev preview, this changes the approach to swap between the "classic" dev preview and Sandpack, with Sandpack being the default. This keeps the docs experience unchanged unless the user specifically opts into the dev preview.